### PR TITLE
Added luqum as a package.

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by thewchan** [comment]
@jaimergp
Appreciate any help you can offer!

File: recipes/luqum/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - luqum
+    - luqum.elasticsearch
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jurismarches/luqum
+  license: Apache-2.0 AND LGPL-3.0-only
```
Comment (isuruf) [inline recipes/luqum/meta.yaml:36]:
```suggestion
  license: Apache-2.0 OR LGPL-3.0-only
```

File: recipes/luqum/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+{% set name = "luqum" %}
+{% set version = "0.11.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.linux-x86_64.tar.gz
```
Comment (isuruf) [inline recipes/luqum/meta.yaml:9]: I think you are better off using the github link.

```suggestion
  url: https://github.com/jurismarches/luqum/archive/refs/tags/{{ version }}.tar.gz
```

File: recipes/luqum/meta.yaml (hunk 1 above)
Comment (thewchan) [reply recipes/luqum/meta.yaml]: Gotcha!

File: recipes/luqum/meta.yaml (hunk 2 above)
Comment (thewchan) [reply recipes/luqum/meta.yaml]: Made this choice and hope it works 🤞

**Comment by thewchan** [comment]
@isuruf @conda-forge/staged-recipes Looks like it's ready for review!

**Pull request merged**